src/Model/GalleryMediaCollectionInterface.php

```diff
@@ -19,7 +19,7 @@
  */
 interface GalleryMediaCollectionInterface
 {
-    public function addGalleryHasMedia(GalleryHasMediaInterface $galleryHasMedia);
+    public function addGalleryHasMedia(GalleryItemInterface $galleryHasMedia);
```

Comment:
Shouldn't the parameter name be updated as well?

Reply:
idk. I just know that like that it is an error.
As I am not too much into SMB I considered keeping the change as small as possible.
As this would have repercussions to other Interfaces, Classes and the like... And as such break functionallity...

Reply:
Changing a parameter name has no consequence whatsoever: https://3v4l.org/HBWVA

Reply:
Done. Sorry. Thought you spoke about the function name 
... which does not seem proper either